src/components/Article.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router";
+import { getArticleById } from "../api";
+
+export const Article = () => {
+  const { id } = useParams();
+  const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getArticleById(id).then(({ data }) => {
+      setArticle(data.article);
+      setIsLoading(false);
+    });
+  }, [id]);
+  
+  if (!isLoading) {
+
+    const topic = article.topic.charAt(0).toUpperCase() + article.topic.slice(1);
+    const date = article.created_at.split("T")[0];
```

Comment:
Have a look at the built in Date constructor to help with formatting dates. Or a library like moment.js 😄